app/scripts/controllers/edit/deploymentConfig.js

```diff
@@ -371,23 +372,20 @@ angular.module('openshiftConsole')
 
       DataService.update("deploymentconfigs", $scope.updatedDeploymentConfig.metadata.name, $scope.updatedDeploymentConfig, $scope.context).then(
         function() {
-          AlertMessageService.addAlert({
-            name: $scope.updatedDeploymentConfig.metadata.name,
-            data: {
-              type: "success",
-              message: "Deployment config " + $scope.updatedDeploymentConfig.metadata.name + " was successfully updated."
-            }
+          NotificationsService.addNotification({
+            type: "success",
+            message: "Deployment config " + $scope.updatedDeploymentConfig.metadata.name + " was successfully updated."
           });
           var returnURL = Navigate.resourceURL($scope.updatedDeploymentConfig);
           $location.url(returnURL);
         },
         function(result) {
           $scope.disableInputs = false;
-          $scope.alerts["save"] = {
+          NotificationsService.addNotification({
```

Comment:
Need an ID on this that is cleared if the edit succeeds or the user cancels the edit.